pytket/docs/changelog.rst

```diff
@@ -4,6 +4,10 @@ Changelog
 0.19.0 (unreleased)
 -------------------
 
+Minor new features:
+
+* Add ``cnx_gray_decomp`` method for C5X, C6X and C7X decomposition.
```

Comment:
This isn't a pytket method so the wording isn't quite right here, but we could say something about improvements to CnX decomposition for n >= 5.